### DISCUSSION TITLE
Request for config that stops torrent upon complete except for one category.

### DISCUSSION THREAD

**Discussion by devanteweary** [StuffAnThings/qbit_manage#373] [Q&A]
Hey everyone,

I've been trying at this for a few days with no success so thought I'd ask here...

All I want is all my torrents to stop except for any in the "Books" category.
I'm not sure why it isn't doing anything but when I run qbit_manage, it just... does nothing.

I'm on Unraid > Docker > qbit_manage 

And a side question.... can I have qbit_manage do its thing on a schedule of my choosing?
Like can I have it stop torrents every 10 minutes?

--------------

Here's there relevant config parts...

commands:
  dry_run: false
  cross_seed: false
  recheck: false
  cat_update: false
  tag_update: false
  rem_unregistered: false
  tag_tracker_error: false
  rem_orphaned: false
  tag_nohardlinks: false
  share_limits: true
  skip_qb_version_check: false
  skip_cleanup: false

directory:
  cross_seed:
  root_dir: /data/

cat:
  Books: /data/Complete/Media/Books
  Console - 3DS: /data/Complete/Console/3DS
  Console - GameCube: /data/Complete/Console/GameCube
  PC - Games: /data/Complete/Games/PC
  PC - Utilities: /data/Complete/PC/Utilities
  Video - Movies: /data/Complete/Media/Video/Movies
  Video - Series: /data/Complete/Media/Video/Series

share_limits:
  KeepBooks:
    priority: 1
    categories:
    - Books
    max_seeding_time: -1
    cleanup: false

  default:
    priority: 999
    max_ratio: -1
    max_seeding_time: 1
    cleanup: false

**Comment by bakerboy448** [comment]
- That's not valid YML and the indents are off
- You've told qbit to do nothing in the YNL and set all commands to false / presumably are not passing any as envars
- Review https://github.com/StuffAnThings/qbit_manage/wiki/Unraid-Installation and https://github.com/StuffAnThings/qbit_manage/wiki/Commands

**Comment by devanteweary** [comment]
> * That's not valid YML and the indents are off
> 
>     * You've told qbit to do nothing in the YNL and set all commands to false / presumably are not passing any as envars
> 
>     * Review [StuffAnThings/qbit_manage/wiki/Unraid-Installation](https://github.com/StuffAnThings/qbit_manage/wiki/Unraid-Installation) and [StuffAnThings/qbit_manage/wiki/Commands](https://github.com/StuffAnThings/qbit_manage/wiki/Commands)

Thanks for the reply and so quickly too!

Those are just the parts of the YML I thought relevant, not the entire thing.
I didn't realize indents matter though!

At the top I have share_limits set to true. 

I've been though the Unraid installation guide and the Config setup guide but sorry, still confused by it.
From what I understand, setting the share_limits to true should only enable that function, right?

Last question! You sent the Commands link...
What's the idea behind running commands while using a Docker container?
Everytime I load the container, I have to go in and run a command? It's not automated?

Sorry about this confusion!

**Reply by bakerboy448** [reply]
Your `share_limits` structure is valid — no schema changes since the earlier reply. Answering your three questions:

**Is `share_limits: true` enough?** Yes, each command flag is independent — you don't need other `commands` keys set to `true` for share-limits to run.

**Will `KeepBooks`/`default` do what you want?** Groups are evaluated in priority order (lower = higher priority), first match wins, one group per torrent. `KeepBooks` (priority 1, `categories: [Books]`) checked before `default` (priority 999, no `categories` = matches everything else) is the right pattern.

One precision note: `max_seeding_time: 1` means "stop 1 minute after seeding starts," not instantly on completion — it's parsed as minutes. Use `0` for closest-to-immediate.

Also add `share_limit_action: Stop` on `default` explicitly — that tells qBittorrent itself to act the moment the limit is hit rather than relying on qBittorrent's own separately-configured default action. See Config-Setup.md → Share Limit Action.

**Scheduling:** `--schedule` / `QBT_SCHEDULE` (default 1440 min = daily). Set a smaller interval or a cron expression; without `-r`/one-shot mode, the container stays running and re-executes on schedule automatically. See Commands.md.

If it's still not acting, share the full config with real indentation (e.g. a gist) plus a `--dry-run` log snippet — that'll show whether torrents are matching a group at all.
